platforms/jvm/plugins-application/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -411,7 +411,7 @@ private String fullMainArgument() {
     }
 
     @Input
-    @ToBeReplacedByLazyProperty
+    @ToBeReplacedByLazyProperty(skipForReport = true, comment = "Skipped for report since method is protected")
```

Comment:
❓ Why wouldn't we report `protected` methods? Types extending this class can still access the method.

Reply:
I see. I guess that's okay.